### PR TITLE
feat: Added public APIs

### PR DESCRIPTION
**Pull request by dhavalarora**
Added requested public APIs.

### CODE REVIEW

File: apps/acqdat_api/lib/acqdat_api/extract_public_routes.ex [hunk 1]
```diff
@@ -16,7 +19,9 @@ defmodule AcqdatApi.ExtractPublicRoutes do
         create_map(route, acc)
       end)
 
-    create_policies(routes)
+    routes
+    |> Map.drop(@neglect_routes)
+    |> create_policies()
```
Comment (BandanaPandey) [inline apps/acqdat_api/lib/acqdat_api/extract_public_routes.ex:24]: @dhavalarora  why did you ignore these routes, is there any specific reason ?

File: apps/acqdat_api/lib/acqdat_api_web/controllers/entity-management/public_api/project_controller.ex [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            metadata(:string, "Project metadata")
+          end
+
+          relationship(:org)
+          relationship(:creator)
+          relationship(:leads, type: :has_many)
+          relationship(:users, type: :has_many)
+        end,
+      Projects: JsonApi.page(:ProjectResource)
+      # Project: JsonApi.single(:ProjectResource)
```
Comment (BandanaPandey) [inline apps/acqdat_api/lib/acqdat_api_web/controllers/entity-management/public_api/project_controller.ex:34]: @dhavalarora Please remove commented lines of code

**Review by BandanaPandey — APPROVED** [review]

File: apps/acqdat_api/lib/acqdat_api/extract_public_routes.ex (hunk 1 above)
Comment (dhavalarora) [reply apps/acqdat_api/lib/acqdat_api/extract_public_routes.ex:24]: Yes, coz that's the route of swagger endpoint that hosts Swagger documentation. We do not need policy for that in MADS.

File: apps/acqdat_api/lib/acqdat_api_web/controllers/entity-management/public_api/project_controller.ex (hunk 2 above)
Comment (dhavalarora) [reply apps/acqdat_api/lib/acqdat_api_web/controllers/entity-management/public_api/project_controller.ex]: Done

**Pull request merged**